content/rs/clusters/maintenance-mode.md

```diff
@@ -156,13 +178,13 @@ After turning on maintenance mode for node 2, Redis Enterprise saves a snapshot
 
 Now node 2 has `0/0` shards because shards cannot migrate to it while it is in maintenance mode.
 
-## Toggle maintenance mode via API
+## Maintenance mode REST API
```

Comment:
Maybe "Change maintenance mode with REST API" or even "Maintenance mode with REST API"? I'm not sure if this is the best heading for this section.

Reply:
Yeah, this went through multiple iterations; unfortunately, many wrapped default sizing.  This one remains on one line.